Jellyfin/dist/presence.js

```diff
@@ -0,0 +1,419 @@
+// official website
+const JELLYFIN_URL = "jellyfin.org";
+
+// web client document.title value (case sensitive)
+const DOCUMENT_TITLE = "Jellyfin";
+
+// web client app name
+const APP_NAME = "Jellyfin Web";
+
+// all the presence art assets uploaded to discord
+const PRESENCE_ART_ASSETS = {
+    download: "downloading",
+    live: "live",
+    logo: "banner-icon",
+    pause: "pause",
+    play: "play",
+    read: "reading",
+    search: "search",
+    write: "writing"
+}
+
+let presence = new Presence({
+    clientId: "669359568391766018", // The client ID of the Application created at https://discordapp.com/developers/applications
+    mediaKeys: false // Enable use and detection of media key presses
+});
+
+// View https://docs.premid.app/en/dev/presence/class -> presenceData interface for more
+let presenceData = {
+    /*The key (file name) of the Large Image on the presence. These are uploaded and named in the Rich Presence section of your application, called Art Assets*/
+    largeImageKey: PRESENCE_ART_ASSETS.logo,
+
+    /*The key (file name) of the Large Image on the presence. These are uploaded and named in the Rich Presence section of your application, called Art Assets*/
+    // smallImageKey: "key",
+
+    //The text which is displayed when hovering over the small image
+    // smallImageText: "Some hover text",
+
+    // The upper section of the presence text, 2nd line
+    // details: "",
+
+    // The lower section of the presence text, 3rd line
+    // state: "",
+
+    // The unix epoch timestamp for when to start counting from
+    // startTimestamp: null,
+
+    // If you want to show Time Left instead of Elapsed, this is the unix epoch timestamp at which the timer ends
+    // endTimestamp: 1577151472000
```

Comment:
Please remove all the comments that are from the docs, please leave only comments about the presence itself or your todos etc.

Reply:
Comments from the docs removed in commit [cdd7d3f7fec4a363fa028dd01fa3fabbe5d2bd76](https://github.com/alexbcberio/Presences/commit/cdd7d3f7fec4a363fa028dd01fa3fabbe5d2bd76)